Casks/k/kunkun.rb

```diff
@@ -0,0 +1,29 @@
+cask "kunkun" do
+  arch arm: "aarch64", intel: "x64"
+
+  version "0.1.22"
+  sha256 arm:   "50c18b43db81f82db893fcef09800d0d0ca59605dbda43ce9875d2027d2216fe",
+         intel: "8b6e77b3c4798ac8c7655b7ec86a12550690c5448ee8769e4f07bca0ef888553"
+
+  url "https://github.com/kunkunsh/kunkun/releases/download/Kunkun-v#{version}/kunkun_#{version}_#{arch}.dmg",
+      verified: "github.com/kunkunsh/kunkun/"
+  name "kunkun"
+  desc "Open source, cross-platform, extensible app launcher"
```

Comment:
```suggestion
  desc "App launcher"
```

Per our [desc stanza guidelines](https://docs.brew.sh/Cask-Cookbook#stanza-desc), we trim the marketing fluff out of descriptions in `homebrew-cask`.